test/app-datepicker-test.html

```diff
@@ -80,6 +80,16 @@
             expect(thirdDay.getAttribute('aria-disabled')).to.equal('');
             expect(fourthDay.getAttribute('aria-disabled')).to.equal('');
           });
+
+          it('should support falsy values', function() {
```

Comment:
It would throw an error before, which makes it so the test fails. An expect wasn't necessary to get the desired effect.